src/index.html

```diff
@@ -8,6 +8,56 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      
```

Comment:
Remove empty line here

Add empty lines between multiline sibling blocks of HTML. We need it to add some "air" and simplify reading. **But don't add them between parent and child elements.** ([checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md))